functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,33 @@
 #include "impl.h"
 
+BeerBrand BeerOrganizer::operator()()
+{
+    if ( mLastBeerBrand == static_cast<size_t>( BeerBrand::Max ) )
+    {
+        mLastBeerBrand = static_cast<size_t>( BeerBrand::None ) + 1;
+    }
+
+    return static_cast<BeerBrand>( mLastBeerBrand++ );
+}
+
+bool isExpensiveBeer( const BeerBrand brand )
+{
+    std::array<BeerBrand,2> expensiveOnes = { BeerBrand::Corona, BeerBrand::HoeGaarden };
+    
+    for ( const BeerBrand expensive : expensiveOnes )
```

Comment:
in your case you can use std::initializer list as iteration range, but I think we don't need this loop